package.json

```diff
@@ -23,4 +24,4 @@
     "typescript": "^4.9.3",
     "vite": "^4.1.0"
   }
-}
+}
```

Comment:
Standard practice is to end each file with exactly one blank line - see how GitHub has added a ⛔ symbol here?